web/src/routes/PluginMain.tsx

```diff
@@ -1,14 +1,16 @@
 import {Banner, Flex, FlexItem, Page} from '@patternfly/react-core';
 
 import {Navigate, Outlet, Route, Routes} from 'react-router-dom';
+import {RecoilRoot} from 'recoil';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
```

Comment:
we use these libs for our state management. Is this already loaded in the main container?